website/common/locales/en/character.json

```diff
@@ -171,5 +171,8 @@
     "quickAllocationLevelPopover": "Each level earns you one point to assign to an attribute of your choice. You can do so manually, or let the game decide for you using one of the Automatic Allocation options found in User -> Stats.",
     "invalidAttribute": "\"<%= attr %>\" is not a valid attribute.",
     "notEnoughAttrPoints": "You don't have enough attribute points.",
-    "gearNotOwned": "You do not own this item."
+    "gearNotOwned": "You do not own this item.",
```

Comment:
Since we're adding a lot of new strings that are not used in the current site me and negue started using a newClient.json file so we don't load them in the old site since there are quite a bit of them now